templates/cli/lib/commands/generic.js.twig

```diff
@@ -68,6 +69,13 @@ const client = new Command("client")
             try {
                 let url = new URL(endpoint);
                 if (url.protocol === "http:" || url.protocol === "https:") {
+                    let apClient = new Client().setEndpoint(endpoint);
+                    let response = await apClient.call('GET', '/health/version', {}, {'x': 'a', 'y': 1, 'z': ['a']});
```

Comment:
Just a question, is `apClient` on purpose or a type and it should be `apiClient`?

Reply:
And we also don't need to send data with it 👍🏻

Reply:
`ap` meaning `appwrite`